generator/generator.yml

```diff
@@ -122,3 +122,21 @@ modules:
       - 1.3.6.1.4.1.25461.2.1.2.3 # panSession
       - 1.3.6.1.4.1.25461.2.1.2.5 # panGlobalProtect
 
+# Arista Networks
+#
+# Arista Networks MIBs can be found here: https://www.arista.com/en/support/product-documentation/arista-snmp-mibs
+#
+# https://www.arista.com/assets/data/docs/MIBS/ARISTA-ENTITY-SENSOR-MIB.txt
```

Comment:
Do I need to change the link in readme with this?
https://www.arista.com/en/support/product-documentation/arista-snmp-mibs